components/module-components/create-blogs/CreateBlogs.tsx

```diff
@@ -0,0 +1,72 @@
+'use client';
+
+import React, { useEffect, useState } from 'react';
+import Button from '../../reusable-components/button/Button';
+import ReusableTable from '../../reusable-components/reusable-table/ReusableTable';
+import Filter from '../SearchAndFilter/Filter';
+import Search from '../SearchAndFilter/Search';
+import style from './createBlogs.module.scss';
+
+interface BlogPost {
+  id: string;
+  title: string;
+  tags: { name: string }[];
+  author: string;
+}
+
+const CreateBlogs = () => {
```

Comment:
We need better wording here. This will be the page where we would display the blogs data. Like a view for the blogs in content-panel